packages/stream_chat_flutter/lib/src/message_input/voice_notes/audio_loading_message.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter/material.dart';
+
+class AudioLoadingMessage extends StatelessWidget {
+  const AudioLoadingMessage({
```

Comment:
Although audio message loading widget needs to be changed but keep it as it is for now.

Reply:
Check with designer and provide loader for replacement.